site/index.md

```diff
@@ -53,6 +53,14 @@ images:
 
 Read our [introduction article on Medium](https://medium.com/p/438f9215f09e), check out the [documentation]({{site.baseurl}}/docs/) and take a look at our [example gallery]({{site.baseurl}}/gallery.html).
 
+## Using Vega-Lite
+
+<!--TODO more about API -->
+
+To get started quickly, you can create visualizations in our [online editor](https://vega.github.io/vega-editor/?mode=vega-lite).
+
+There are many ways to use Vega-Lite in your webpage or project. You can use the included compile function, the bundled command-line tools, our easy-to-use [embed library](https://github.com/vega/vega-embed), or a multitude of third party tools. To learn more about using Vega-Lite and other methods of compiling Vega-Lite specifications, visit our [usage instructions]({{site.baseurl}}/docs/usage.html).
```

Comment:
Can you compress this? Also, put a link to command line tools section of the usage instructions.